app/org/sagebionetworks/bridge/dynamodb/DynamoThroughput.java

```diff
@@ -0,0 +1,13 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface DynamoThroughput {
+    long writeCapacity() default DynamoInitializer.WRITE_CAPACITY;
+    long readCapacity() default DynamoInitializer.READ_CAPACITY;
+}
```

Comment:
There's already defaulting logic in DynamoInitializer. I feel like having defaults here would be redundant. Consider making read and write capacity required. Also, consider shortening them to write() and read() so we can write the annotation shorter:
@DynamoThroughput(read = 50, write = 50)